src/find-options/FindOptionsUtils.ts

```diff
@@ -222,7 +222,7 @@ export class FindOptionsUtils {
     /**
      * Adds joins for all relations and sub-relations of the given relations provided in the find options.
      */
-    protected static applyRelationsRecursively(qb: SelectQueryBuilder<any>, allRelations: string[], alias: string, metadata: EntityMetadata, prefix: string): void {
+    public static applyRelationsRecursively(qb: SelectQueryBuilder<any>, allRelations: string[], alias: string, metadata: EntityMetadata, prefix: string): void {
```

Comment:
Is there any way we can keep this private

Reply:
I don't really see a way of using this function without being public. It is used in another class, so it has to be public if it should stay in the FindOptionsUtils class.

Reply:
Why would it be important that the function stays protected?

Reply:
Limit the pubic API footprint.

As soon as it becomes public we have to support it as a part of the API

Reply:
Okay.. is there something I can do now? Because making it non public is not possible, the only option would be to copy the method whoch leads to duplicate code. Or do you have another idea?

Reply:
making it public is better than code duplication